evalml/automl/automl_algorithm/default_algorithm.py

```diff
@@ -326,8 +289,14 @@ def _create_n_pipelines(self, pipelines, n):
                     self._create_tuner(pipeline)
 
                 select_parameters = self._create_select_parameters()
-                proposed_parameters = self._tuners[pipeline.name].propose()
-                parameters = self._transform_parameters(pipeline, proposed_parameters)
+                parameters = (
+                    self._tuners[pipeline.name].get_starting_parameters(
+                        self._hyperparameters, self.random_seed
+                    )
+                    if n == 1
```

Comment:
I think we want `_batch_number <= 2` here right? I think this works right now because we only call `_create_n_pipelines` with `n=1` in the create_fast_final batch but would not work if the code was refactored I think.

Reply:
a good alternative here would be to refactor this logic to be under a flag instead of basing it off of the number of pipelines generated

Reply:
@jeremyliweishih great suggestion! I'll change that.